payment_processors/braintree_processor/models/payment_processors.py

```diff
@@ -6,12 +6,11 @@
 from django.utils import timezone
 from django_fsm import TransitionNotAllowed
 
+from .payment_methods import BraintreePaymentMethod
+from ..views import BraintreeTransactionView
```

Comment:
http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module